plugins/rtmp-services/data/services.json

```diff
@@ -1684,11 +1684,17 @@
             ]
         },
         {
-            "name": "niconico, premium member (ニコニコ生放送 プレミアム会員)",
+            "name": "niconico(ニコニコ生放送)",
```

Comment:
This name change eliminates a space before the opening parenthesis while the previous names had this space. If this is intentional, that's fine.

Reply:
Thank you for the comment! I've adjusted it to match the existing formatting style.